redbot/cogs/general/general.py

```diff
@@ -234,38 +240,210 @@ def __init__(self):
-        data.add_field(name=_("Text Channels"), value=text_channels)
-        data.add_field(name=_("Voice Channels"), value=voice_channels)
-        data.add_field(name=_("Roles"), value=humanize_number(len(guild.roles)))
-        data.add_field(name=_("Owner"), value=str(guild.owner))
-        data.set_footer(text=_("Server ID: ") + str(guild.id))
-
-        if guild.icon_url:
-            data.set_author(name=guild.name, url=guild.icon_url)
-            data.set_thumbnail(url=guild.icon_url)
+        if not details:
+            data = discord.Embed(description=created_at, colour=await ctx.embed_colour())
+            data.add_field(name=_("Region"), value=str(guild.region))
+            data.add_field(name=_("Users"), value=f"{online}/{total_users}")
+            data.add_field(name=_("Text Channels"), value=text_channels)
+            data.add_field(name=_("Voice Channels"), value=voice_channels)
+            data.add_field(name=_("Roles"), value=humanize_number(len(guild.roles)))
+            data.add_field(name=_("Owner"), value=str(guild.owner))
+            data.set_footer(text=_("Server ID: ") + str(guild.id))
+            if guild.icon_url:
+                data.set_author(name=guild.name, url=guild.icon_url)
+                data.set_thumbnail(url=guild.icon_url)
+            else:
+                data.set_author(name=guild.name)
         else:
-            data.set_author(name=guild.name)
 
-        try:
-            await ctx.send(embed=data)
-        except discord.Forbidden:
-            await ctx.send(_("I need the `Embed links` permission to send this."))
+            def _size(num: int):
+                for unit in ["B", "KB", "MB", "GB", "TB", "PB", "EB", "ZB"]:
+                    if abs(num) < 1024.0:
+                        return "{0:.1f}{1}".format(num, unit)
+                    num /= 1024.0
+                return "{0:.1f}{1}".format(num, "YB")
+
+            def _bitsize(num: int):
+                for unit in ["B", "KB", "MB", "GB", "TB", "PB", "EB", "ZB"]:
+                    if abs(num) < 1000.0:
+                        return "{0:.1f}{1}".format(num, unit)
+                    num /= 1000.0
+                return "{0:.1f}{1}".format(num, "YB")
+
+            # Logic from: https://github.com/TrustyJAID/Trusty-cogs/blob/master/serverstats/serverstats.py#L159
+            online_stats = {
+                _("Humans: "): lambda x: not x.bot,
+                _(" • Bots: "): lambda x: x.bot,
+                "📗": lambda x: x.status == discord.Status.online,
+                "📙": lambda x: x.status == discord.Status.idle,
+                "📕": lambda x: x.status == discord.Status.idle,
+                "📓": lambda x: x.status == discord.Status.offline,
+                "🎥": lambda x: x.activity == discord.Streaming,
+                "📱": lambda x: x.is_on_mobile(),
+            }
+            member_msg = _("Total Users: {}\n").format(bold(f"{online}/{total_users}"))
```

Comment:
I think it's gonna be clearer to call this "Users online". I also put `{online}/{total_users}` cause it might help translators to know the context.
```suggestion
            member_msg = _("Users online: **{online}/{total_users}**\n").format(
                online=online, total_users=total_users
            )
```